contracts/cw1-subkeys/src/contract.rs

```diff
@@ -43,62 +44,63 @@ impl Cw1SubkeysContract<'_> {
     #[msg(instantiate)]
     pub fn instantiate(
         &self,
-        ctx: (DepsMut, Env, MessageInfo),
+        mut ctx: InstantiateCtx,
         admins: Vec<String>,
         mutable: bool,
     ) -> Result<Response, ContractError> {
-        let (mut deps, env, info) = ctx;
-        let result = self
-            .whitelist
-            .instantiate((deps.branch(), env, info), admins, mutable)?;
-        set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
+        let result = self.whitelist.instantiate(ctx.branch(), admins, mutable)?;
+        set_contract_version(ctx.deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
         Ok(result)
     }
 
     #[msg(exec)]
     pub fn increase_allowance(
         &self,
-        ctx: (DepsMut, Env, MessageInfo),
+        ctx: ExecCtx,
         spender: String,
         amount: Coin,
         expires: Option<Expiration>,
     ) -> Result<Response, ContractError> {
-        let (deps, env, info) = ctx;
         ensure!(
-            self.whitelist.is_admin(deps.as_ref(), &info.sender),
+            self.whitelist.is_admin(ctx.deps.as_ref(), &ctx.info.sender),
             ContractError::Unauthorized {}
         );
 
-        let spender = deps.api.addr_validate(&spender)?;
-        ensure_ne!(info.sender, spender, ContractError::CannotSetOwnAccount {});
-
-        self.allowances.update(deps.storage, &spender, |allow| {
-            let prev_expires = allow
-                .as_ref()
-                .map(|allow| allow.expires)
-                .unwrap_or_default();
-
-            let mut allowance = allow
-                .filter(|allow| !allow.expires.is_expired(&env.block))
-                .unwrap_or_default();
+        let spender = ctx.deps.api.addr_validate(&spender)?;
+        ensure_ne!(
+            ctx.info.sender,
+            spender,
+            ContractError::CannotSetOwnAccount {}
```

Comment:
Why use `{}` for errors? They don't need it; they are not serialized.

Reply:
Agree. This is unnecessary

Reply:
Will deliver fix in separate PR

Reply:
Removal of curly braces on errors #95